src/components/Absence/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { Absence } from "../../types";
+import EmployeeItem from "../Employee";
+import { format, addDays } from "date-fns";
+
+type Props = {
+  absence: Absence;
+};
+
+const AbsenceItem = ({ absence }: Props) => {
+  return (
+    <div>
```

Comment:
nice way of structuring this, but you could make an improvement by structuring this data in a table, to give a tabular view, which helps with sorting and filtering functionality (down the line)